src/routes/availability/[slug]/+page.server.ts

```diff
@@ -15,24 +15,30 @@ import {
   type AvailabilityInsertSchema,
   type AvailabilityMeetingDateJoinSchema,
   type MeetingDateSelectSchema,
+  users,
+  meetings,
+  members,
 } from "$lib/db/schema";
 import type { ZotDate } from "$lib/utils/ZotDate";
 
 export const load: PageServerLoad = (async ({ locals, params }) => {
   const user = locals.user;
+  // @ts-expect-error slug is defined in the route
+  const meeting_id: string = params?.slug;
```

Comment:
```suggestion
  const meeting_id: string = params?.slug ?? "";
```
chore: unused ts directive -- slug is read as `any`. It might be better to use the nullish coalescing operator `??` to force it into a string